ims.md

```diff
@@ -0,0 +1,43 @@
+# IMS - What is it?
+
+IMS stands for **Integrated Management System**. It combines all the overarching systems that Made Tech use to govern our business operations. More specifically, our IMS currently manages our Security and Quality systems which have been implemented as part of our work towards our certification in ISO 27001 and ISO9001. 
+
+## ISO 27001
+### What is ISO 27001?
+ISO 27001 is an internationally recognised standard in Security Management. 
+### Why is Made Tech implementing ISO 27001?
+Security is an important part of any organisation. It not only ensures that we are a trustworthy business for our clients and partners to work with, but also ensures that we are a responsible employer. By implementing and adhering to these security policies, we ensure:
+
+1) That all members of the Made Tech team can be confident that the company is handling their personal information appropriately.
+2) Our clients and partners can be confident that the availability, integrity and confidentiality of their information will be maintained appropriately.
+
+These security policies also:
+ - Protect Made Tech’s ability to meet commitments through Business Continuity planning.
+ - Minimise the business impact of, and appropriately handle, security events and incidents.
+
+The relevant people in the business are
+- Chief Security Officer (CSO): Chris Blackburn
+- IMS Manager: Emily Thomas
+ 
+## ISO 9001
+### What is ISO 9001
+ISO 9001 is an internationally recognised standard in Quality Management
+### Why is Made Tech implementing ISO 9001
+Goods and Services can have varying degrees of quality. Made Tech's mission is "Improving software delivery in every organisation", and a large part of that is endeavouring to deliver high quality products that both meet our clients expectations, and meet the needs of real users. The policies implemented in line with ISO 9001 help us to acheive that goal. 
+
+The relevant people in the business are
+- Quality Officer: Chris Blackburn/Luke Morton (TBD)
+- IMS Manager: Emily Thomas
+
+# Guide
```

Comment:
I think this should be a level 2 header, and then level 3 headers underneath it?